### PR TITLE
dopamine: Persist settings file (Solve #4327)

### LINKED ISSUE #4327: Dopamine settings not persisted
Next time opening Dopamine after updating it always starts with the configuration screen.  
Looks like some settings aren't getting persisted

### PR DESCRIPTION
**Pull request by Rafeqm**
Closes #4327

- [x] I have read the [Contributing Guide](https://github.com/ScoopInstaller/.github/blob/main/.github/CONTRIBUTING.md).

### CODE REVIEW

File: bucket/dopamine.json [hunk 1]
```diff
@@ -11,6 +11,7 @@
             "Dopamine"
         ]
     ],
+    "persist": "Dopamine\\Settings.xml",
```
Comment (rashil2000) [inline bucket/dopamine.json:14]: On a fresh installation, does this file already exist? Or is it created later?

Comment (Rafeqm) [reply bucket/dopamine.json:14]: Could be both. I don't know cause I haven't intalled it yet. If the file is created later, can scoop create that file? Of course, the file content will be edited by the app itself, or in worse scenario the file will be tampered with the new settings applied.
The first time I know scoop can persist _file_ (not just folder) is after installing Jetbrains IDEs via scoop. It also the first time I installed those IDEs.

Comment (rashil2000) [reply bucket/dopamine.json:14]: If the file doesn't exist on a fresh installation, Scoop will assume the item is a _folder_ and create a folder hard link. To create the file, you can do something like this: https://github.com/ScoopInstaller/Extras/blob/943b50ec4541a630533d695a25edf1f54e4d65f3/bucket/q-dir.json#L39

Comment (Rafeqm) [reply bucket/dopamine.json:14]: What is that `start.qdr`?

File: bucket/dopamine.json [hunk 2]
```diff
@@ -5,6 +5,7 @@
     "license": "GPL-3.0-only",
     "url": "https://www.digimezzo.com/content/software/dopamine/Dopamine%202.0.9%20(Release)%20-%20Portable.zip",
     "hash": "fd5ec9aa1c6f9a2a21244a65f9d77ef20614422c9ce6df75614ba8edb09c8b08",
+    "post_install": "if (!(Test-Path \"$persist_dir\\Dopamine\\Settings.xml\")) { New-Item \"$dir\\Dopamine\\Settings.xml\" | Out-Null }",
```
Comment (rashil2000) [inline bucket/dopamine.json:8]:
```suggestion
    "pre_install": "if (!(Test-Path \"$persist_dir\\Dopamine\\Settings.xml\")) { New-Item \"$dir\\Dopamine\\Settings.xml\" | Out-Null }",
```

Comment (Rafeqm) [reply bucket/dopamine.json:8]: Umm isn't it got checked after installation?

Comment (rashil2000) [reply bucket/dopamine.json:8]: Hard link is created before post_install. So we need to create the file before the hard link creation.

**Pull request merged**